frontend/src/container/ListOfDashboard/utils.ts

```diff
@@ -0,0 +1,18 @@
+export const filterDashboard = (
+	searchValue: string,
+	dashboardList: any,
+): any[] => {
+	// Convert the searchValue to lowercase for case-insensitive search
+	const searchValueLowerCase = searchValue.toLowerCase();
+
+	// Use the filter method to find matching objects
+	return dashboardList.filter((item: any) => {
+		// Convert each property value to lowercase for case-insensitive search
+		const itemValues = Object.values(item?.data).map((value: any) =>
```

Comment:
Once the above `any` is replaced with `Dashboard`, no need to `(value: any)`, can be replaced by `(value)`, as the typescript will infer it internally.